src/contracts/middleware/MiddlewareStorage.sol

```diff
@@ -32,6 +33,7 @@ abstract contract MiddlewareStorage {
     uint8 public constant DEFAULT_DECIMALS = 18;
     uint256 public constant VERSION = 1;
     uint256 public constant PARTS_PER_BILLION = 1_000_000_000;
+    uint256 public constant MAX_BLOCK_INTERVAL = 50; // 600 seconds ≈ 10 minutes
```

Comment:
This could have better name. I think it should be MIN for starters. 
How about `MIN_INTERVAL_TO_SEND_OPERATOR_KEYS`?